pkg/reconciler/reconciler.go

```diff
@@ -401,6 +401,15 @@ func Run(opts Options) {
 		}
 	}
 
+	// Register the Rsync Status Watch Controller
+	rsyncWatchLogger := opts.Logger.WithName("controllers").WithName("rsync-status-watch-controller")
```

Comment:
Looks like there was some confusion maybe. I think we agreed to have one shared controller in the reconciler-manager, not one in each of the reconcilers.

Reply:
Done